analysis_service/base/models.py

```diff
@@ -0,0 +1,206 @@
+from datetime import datetime, timedelta
+from pytz import UTC
+from django.db import models
+from django.contrib.auth.models import User
+
+from analysis_service.base.util import provisioning, scheduling
+
+
+class Cluster(models.Model):
```

Comment:
That could potentially work for clusters, but not for scheduled jobs, which aren't associated with any particular jobflow. Might make testing a bit messier, though.